test/unit_tests/braket/tasks/test_local_quantum_task.py

```diff
@@ -13,17 +13,17 @@
 
 import uuid
 
-import numpy as np
 import pytest
 
 from braket.task_result import TaskMetadata
 from braket.tasks import GateModelQuantumTaskResult
 from braket.tasks.local_quantum_task import LocalQuantumTask
+from braket.tasks.measurement_list import MeasurementsList
 
 RESULT = GateModelQuantumTaskResult(
     task_metadata=TaskMetadata(**{"id": str(uuid.uuid4()), "deviceId": "default", "shots": 100}),
     additional_metadata=None,
-    measurements=np.array([[0, 1], [1, 0]]),
+    measurements=MeasurementsList([[0, 1], [1, 0]]),
```

Comment:
Curious- do the tests pass without updating this? Since we subclassed ndarray, does `__eq__` still work between the classes as before?